src/gui/mzroll/isotopedialog.cpp

```diff
@@ -62,6 +62,14 @@ IsotopeDialog::IsotopeDialog(MainWindow* parent) : QDialog(parent) {
                     ->hitEvent("Peak Detection",
                                "Isotope Detection Switched",
                                state);
+                if (checked) {
+                    if (!D2Labeled_BPE->isChecked()
+                        && !C13Labeled_BPE->isChecked()
+                        && !N15Labeled_BPE->isChecked()
+                        && !S34Labeled_BPE->isChecked())
+                        C13Labeled_BPE->setChecked(true);
+                } else
+                    C13Labeled_BPE->setChecked(false);
```

Comment:
Maybe we do not need this `else` part at all. Or if we are unchecking C13 checkbox in the `else` condition, then we should uncheck them all. Can you make the change?